apis/dataprotection/v1alpha1/backup_types.go

```diff
@@ -81,6 +81,10 @@ type BackupStatus struct {
 	// +optional
 	PersistentVolumeClaimName string `json:"persistentVolumeClaimName,omitempty"`
 
+	// logFilePersistentVolumeClaimName saves the logfile backup data.
+	// +optional
+	LogFilePersistentVolumeClaimName string `json:"logFilePersistentVolumeClaimName,omitempty"`
+
```

Comment:
The BackupLog below in the ManifestsStatus can be refactored to BackupTimeline or BackupTimeRange

Reply:
yes, the semantics of the `BackupLog` field are not accurate enough, and this will be uniformly refactored in a dedicated PR